build.sbt

```diff
@@ -145,7 +145,7 @@ lazy val tests = libraryProject("tests")
     description := "Tests for core project",
     startYear := Some(2013),
   )
-  .dependsOn(core, testing % "test->test")
+  .dependsOn(core, client, blazeClient, blazeServer, theDsl, testing % "test->test")
```

Comment:
In the medium term, blaze is moving out of this repo, but ember is staying.  It would be better to use Ember here.